ckeditor/static/ckeditor/ckeditor/config.js

```diff
@@ -4,7 +4,18 @@
  */
 
 CKEDITOR.editorConfig = function( config ) {
-	// Define changes to default configuration here. For example:
-	// config.language = 'fr';
-	// config.uiColor = '#AADC6E';
+    // Define changes to default configuration here. For example:
+    // config.language = 'fr';
+    // config.uiColor = '#AADC6E';
+    config.extraPlugins = 'notification';
+    config.extraPlugins = 'autosave';
```

Comment:
I don't think we want to add this here. We don't always want autosave, just on instance we specify